Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftRequestController.cs

```diff
@@ -137,7 +137,10 @@ public class OpenShiftRequestController : Controller
 
             // Step 1c.
             var allRequiredConfigurations = await this.utility.GetAllConfigurationsAsync().ConfigureAwait(false);
-            var kronosTimeZoneId = this.appSettings.KronosTimeZone;
+            var teamMappings = await this.teamDepartmentMappingProvider.GetMappedTeamDetailsAsync(teamsId).ConfigureAwait(false);
+            var teamMapping = teamMappings.FirstOrDefault();
+
+            var kronosTimeZoneId = string.IsNullOrEmpty(teamMapping.KronosTimeZone) ? this.appSettings.KronosTimeZone : teamMapping.KronosTimeZone;
```

Comment:
should this be string.IsNullOrEmpty(teamMapping?.KronosTimeZone) with the ?

Reply:
The mappings between a team and a Kronos organisation job are fundamental to the working of the connector, so there should never be a time when no mappings are returned, however, it is a fair comment